### ISSUE TITLE
Settings editor should have Accessibility TOC entry

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#47626]
Testing -217636

* open settings editor
* make sure TOC is visible
* it has no accessibility entry which IMO it should have

<img width="928" alt="Screenshot 2024-06-25 at 09 56 01" src="https://github-com/user-attachments/assets/61b18a20-7b58-4a3b-ab79-e886b850300f">
